### DISCUSSION TITLE
How to get a header value, without ezdxf changing/fixing it first

### DISCUSSION THREAD

**Discussion by velle** [mozman/ezdxf#1024] [Q&A]
I was investigating how to do unit testing on some generated dxf files (example file here: [frominkscape.dxf.zip](https://github.com/mozman/ezdxf/files/14076363/frominkscape.dxf.zip) ). For instance, I simply want to get the value of `$ACADVER` for a given file, and I was going to use ezdxf for extracting that value. 

    $ python -c "import ezdxf; doc = ezdxf.readfile('frominkscape.dxf'); print(doc.dxfversion)"
    AC1015
    $ python -c "import ezdxf; doc = ezdxf.readfile('frominkscape.dxf'); print(doc.header.get('\$ACADVER'))"
    AC1015

However, the actual value in the file seems to be AC1014:

    $ head -n 8 frominkscape.dxf
      0
        SECTION
      2
    HEADER
      9
    $ACADVER
      1
    AC1014

Could you please explain the intention/logic behind changing this to AC1015? 

Will it do so for any AC1014 file, or just for some?

Can you recommend any tool/method for extracting the value of the header as is?

Sincerely

**Comment by mozman** [comment]
from the `README.md` file

- read/write/new support for DXF versions: R12, R2000, R2004, R2007, R2010, R2013 and R2018
- additional read-only support for DXF versions R13/R14 (upgraded to R2000)
- additional read-only support for older DXF versions than R12 (upgraded to R12)

and the version strings for most DXF versions:
```python
DXF9 = "AC1004"
DXF10 = "AC1006"
DXF12 = "AC1009"
DXF13 = "AC1012"
DXF14 = "AC1014"
DXF2000 = "AC1015"
DXF2004 = "AC1018"
DXF2007 = "AC1021"
DXF2010 = "AC1024"
DXF2013 = "AC1027"
DXF2018 = "AC1032"
```
